app/static/src/app/components/run/RunPlot.vue

```diff
@@ -4,16 +4,17 @@
       :placeholder-message="placeholderMessage"
       :end-time="endTime"
       :plot-data="allPlotData"
-      :solutions="solution ? [solution] : []">
+      :solutions="solution ? [solution, allFitData] : []">
```

Comment:
I wonder if we should rename this `prop` now that it might contains things other than solutions. I originally called this something like 'redrawOnUpdate', or could use e.g. 'redrawTriggers` or `redrawWatches`.

Reply:
sounds good, shall I do that in this PR?

Reply:
Might as well, shouldn't be too big a change